public/js/model/game.js

```diff
@@ -66,6 +66,22 @@ Game.prototype.accumulate = function () {
   return this.cumulativeScore;
 };
 
+Game.prototype.isOver = function () {
```

Comment:
isOver is good to control the game scope :) This is a good thought and strategy in your implementation :)